core/modules/compare/components/Product.ts

```diff
@@ -0,0 +1,22 @@
+import Product from '@vue-storefront/store/types/product/Product'
+
+export const CompareProduct = {
+  name: 'CompareProduct',
+  computed: {
+    isOnCompare () : boolean {
```

Comment:
aren't all products passed to this component on compare? isn't this property useless?

Reply:
+ can we move it to vuex getter? I see a lot of use cases . I'll create task for isOnCart and isOnWishlist too

So we will end up with this.$store.getters.isOnCompare(product)

Reply:
Created getter `isOnCompare`, I've used arrow functions there. Is it ok for you?
```
const getters: GetterTree<CompareState, RootState> = {
  isEmpty: (state) => state.items.length === 0,
  isOnCompare: (state) => (product) => state.items.find(p => p.sku === product.sku)
}
```